setup.py

```diff
@@ -51,9 +51,18 @@
         "Unidecode==1.1.1",
         "Werkzeug==0.16.0",
         "WTForms==2.2.1",
+        "esprit @ git+https://github.com/CottageLabs/esprit.git@edda12177effa0945d99302f0d453b22503e335b#egg=esprit",
+        "dictdiffer @ git+https://github.com/CottageLabs/dictdiffer.git@cc86c1ca1a452169b1b2e4a0cb5fc9e6125bc572#egg=dictdiffer",
+        "flask-swagger @ git+https://github.com/DOAJ/flask-swagger.git@f1dbf918d9903a588eed3cce2a87eeccc9f8cc0e#egg=flask-swagger",
+        "combinatrix @ git+https://github.com/CottageLabs/combinatrix.git@740d255f0050d53a20324df41c08981499bb292c#egg=combinatrix"
```

Comment:
@Steven-Eardley might be worth putting this in the test section, as it's not required for production

Reply:
good thinking